bin/install-winston2.js

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+const fs = require('fs')
+const path = require('path')
+const spawn = require('child_process').spawn
+
+
+if(fs.existsSync(path.join(__dirname,'..','test','winston-2','node_modules','winston'))){
+  process.exit(0)
+}
+
+let proc;
+if(process.env.npm_execpath){
+    proc = spawn(process.env.NODE||'node',[process.env.npm_execpath,'install'],{cwd:path.join(__dirname,'..','test','winston-2'),stdio:'inherit'})
+} else {
+    proc = spawn('npm',['install'],{cwd:path.join(__dirname,'..','test','winston-2'),stdio:'inherit'})
+}
+proc.on('close',(code)=>{
+  process.exit(code||0)
+})
```

Comment:
nit: new line.

BTW, `gts fix` should automatically fix these. Oh wait, not for this file because it is not really in the src/ directory, but it should work for all the other instances of this in the PR. I wonder why `gts check` is passing though.

Reply:
Fair enough. Or a bug/regression in he formatter *rolls up sleeves*.